wordpress-mobile/WordPress-iOS
# Add a CI job to build a special version for translation reviewers

This PR adds a CI job triggered by an API call that builds a custom version of the app for translations reviewers.
The build process downloads all the pending translations, merges them into the current ones, builds the app and uploads it to Appetize.

To test:
1. Get a CircleCI personal token.
2. From the terminal, run the following curl command to start the build:
```
curl -u <token>: -X POST --header "Content-Type: application/json" -d '{
 "branch": "translation-review-builds",
 "parameters": {
   "translation_review_build": true
 }
}' https://circleci.com/api/v2/project/github/wordpress-mobile/WordPress-iOS/pipeline
```
3. Verify that the build finishes without errors.
4. Go to Appetize.io and verify that the app runs as expected.

**Note**: Before merging this, a new version of the release-toolkit has to be released and the reference in this PR updated. Related PR: wordpress-mobile/release-toolkit#133.

PR submission checklist:
